src/main/resources/assets/gregtech/lang/en_US.lang

```diff
@@ -159,8 +159,8 @@ achievement.Apatite=Find Apatite Ore
 achievement.Apatite.desc=Height: 40-60, Chance: 60, Overworld
 achievement.Phosphorus=Find Phosphorus Ore
 achievement.Phosphorus.desc=Height: 40-60, Chance: 60, Overworld
-achievement.Phosphate=Find Phosphate Ore
-achievement.Phosphate.desc=Height: 40-60, Chance: 60, Overworld
+achievement.Pyrochlore=Find Pyrochlore Ore
```

Comment:
You can't remove/rename the old one, it throws errors about invalid achient data on player log in if so. Please fix this.

Reply:
Fuck this IPhone, I misspelled Achivement wrong both times.